@xen-orchestra/web/src/components/hosts/panel/card/HostInfoCard.vue

```diff
@@ -0,0 +1,163 @@
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('started') }}</template>
+        <template #value>{{ relativeStartTime }}</template>
+        <template #addons>
+          <VtsCopyButton :value="relativeStartTime" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('patches') }}</template>
+        <template v-if="!missingPatch.length" #value>
+          <UiInfo accent="success">{{ t('patches-up-to-date') }}</UiInfo>
+        </template>
+        <template v-else #value>
+          <UiInfo accent="warning">{{ t('n-missing', missingPatch.length) }}</UiInfo>
+        </template>
+        <template #addons>
+          <VtsCopyButton :value="missingPatch.length ? t('n-missing', missingPatch.length) : t('patches-up-to-date')" />
+        </template>
+      </VtsCardRowKeyValue>
+    </div>
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import { useXoHostUtils } from '@/composables/xo-host.composable'
+import { useXoHostMissingPatchesCollection } from '@/remote-resources/use-xo-host-missing-patches-collection'
+import type { XoHost } from '@/types/xo/host.type'
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsCopyButton from '@core/components/copy-button/VtsCopyButton.vue'
+import VtsEnabledState from '@core/components/enabled-state/VtsEnabledState.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsObjectIcon from '@core/components/object-icon/VtsObjectIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import UiInfo from '@core/components/ui/info/UiInfo.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiObjectLink from '@core/components/ui/object-link/UiObjectLink.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { host } = defineProps<{
+  host: XoHost
+}>()
+
+const { t } = useI18n()
+
+const { getHostState, getIsMaster, getMasterHostFromPoolId, getPool, getPowerState, getRelativeStartTime } =
+  useXoHostUtils()
+
+const { hostMissingPatches: missingPatch } = useXoHostMissingPatchesCollection({}, () => host.id)
+
+const pool = getPool(host.$pool)
+const relativeStartTime = getRelativeStartTime(host.startTime)
+
+const masterHost = computed(() => getMasterHostFromPoolId(host.$pool))
+const powerState = computed(() => getPowerState(host.power_state))
+</script>
+
+<style scoped lang="postcss">
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+
+  .power-state,
+  .pool-name,
+  .primary-host {
+    display: flex;
+    align-items: center;
+    gap: 0.8rem;
+  }
+}
+</style>
```

Comment:
```suggestion
<style scoped lang="postcss">
.card-container {
  gap: 1.6rem;

    .content {
      display: flex;
      flex-direction: column;
      gap: 0.4rem;
    
      .power-state,
      .pool-name,
      .primary-host {
        display: flex;
        align-items: center;
        gap: 0.8rem;
      }
  }
}
</style>
```

you can add on every `UiCard` in the side panel